big_tests/tests/presence_SUITE.erl

```diff
@@ -343,19 +342,18 @@ versioning(Config) ->
         Stanza = escalus_stanza:roster_add_contact(Bob, bobs_default_groups(),
                                                    bobs_default_name()),
         escalus:send(Alice, Stanza),
-        Received = escalus:wait_for_stanzas(Alice, 2),
+        Received = escalus:wait_for_stanzas(Alice, 3),
 
         escalus:assert_many([is_roster_set, is_iq_result], Received),
 
-        RosterSet = hd(Received),
+        RosterSet = hd([R || R <- Received, escalus_pred:is_roster_set(R)]),
```

Comment:
I think we can assume there is only one `roster set` stanza somweher on the list. How about using a filter like in the other test case or sth like the following:
```suggestion
        [RosterSet] = [R || R <- Received, escalus_pred:is_roster_set(R)],
```